### DISCUSSION TITLE
Example repo wanted: Using Excel for meta-data

### DISCUSSION THREAD

**Discussion by christoph-LSN** [open-sdg/open-sdg#1950] [Q&A] [answered]
@LucyGwilliamAdmin @brockfanning @MoBosse 

Hello, 

for an further project I am looking for a demo repo where Excel is used for meta-data

can you help me?

**Comment by sarah-beasley** [comment]
Hi @christoph-LSN , this is something that the UK currently do - we start with our data and metadata together in an Excel file. Then when we want to update an indicator on the site, we run these Excel files through a python script which converts the data into a .csv file, and the metadata into a .md file. You can see these in our data and meta folders in our data repo [here](https://github.com/ONSdigital/sdg-data). Hope this helps, but let me know if you'd like anything else!

**Comment by christoph-LSN** [comment]
@sarah-beasley Thank you for yuur quick response!
I read in the docs tha it's easy maintaining the metadata in Ecxel. see here: https://open-sdg.readthedocs.io/en/latest/metadata-file-formats/#excel-and-csv

But I don't know how a complete metadata-file in Excel looks like. 
I found this in the kyrgyzstan data repo https://github.com/sdg-kyrgyzstan/open-sdg-data-starter/blob/develop/meta/1-1-1.xlsx but it is in Cyrillic that I cannot read....

Another point is that  I not really understand how to adjust the - class: InputYamlMdMeta  here: https://github.com/christoph-LSN/IM_Data/blob/88c465770bcc9a004a9c374a63a6cb779806bde9/config_data.yml#L44
 

By the way I am very interested in your python script for converting excel to cvs. I wrote a vba script to do this in excel but, its quite slow because sometimes I have to process more than 25k lines in excel

**Comment by brockfanning** [comment, marked as answer]
@christoph-LSN As far as I know, the Excel file should have 2 columns -- the first column as the field and the second column is the value.

To change your data configuration file you would use `class: InputExcelMeta`. There is a bit more information about these Input classes (including InputExcelMeta) here: https://open-sdg.readthedocs.io/en/latest/data-configuration/#inputs

**Comment by christoph-LSN** [comment]
Thank You @brockfanning ! I will have a look at this in a couple of days
